code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -18,11 +18,14 @@
 //Hunter verbs
 
 /mob/living/carbon/alien/humanoid/hunter/proc/toggle_leap(message = 1)
+	if(pounce_cooldown)

```

Comment:
this is the toggle @Robustin, you should be able to freely toggle between the two while fatigued.